app/scripts/components/home.js

```diff
@@ -41,6 +54,14 @@ class Home extends React.Component {
     this.cancelInterval = interval(() => {
       this.query();
     }, updateInterval, true);
+    const {dispatch} = this.props;
+    dispatch(getCumulusInstanceMetadata())
+      .then(() => {
+        dispatch(getDistApiGatewayMetrics(this.props.cumulusInstance));
+        dispatch(getDistApiLambdaMetrics(this.props.cumulusInstance));
+        dispatch(getDistS3AccessMetrics(this.props.cumulusInstance));
+      }
+    );
```

Comment:
I did this to try to make sure the stackName was populated before dispatching the calls to ES, but it seems that sometimes I see the dashboard with zero values across the board before the  buttons update with the correct values.  If there's a better way, please tell me.